src/node.cc

```diff
@@ -1159,6 +1153,32 @@ Maybe<bool> ProcessEmitDeprecationWarning(Environment* env,
                                    deprecation_code);
 }
 
+static void OnMessage(Local<Message> message, Local<Value> error) {
+  Isolate* isolate = Isolate::GetCurrent();
+  switch (message->ErrorLevel()) {
+    case Isolate::MessageErrorLevel::kMessageWarning: {
```

Comment:
IMO whis should come with a CLI arg. opt-in or opt-out, IDK...

Reply:
my understanding of warnings is that you can do
```js
process.on('warning', (warning) => {
  if (warning.name === 'V8') { ... }
});
```

Reply:
Ack.